control-plane/src/modules/router.ts

```diff
@@ -247,4 +266,40 @@ export const router = s.router(contract, {
       body: cluster,
     };
   },
+  getFunctionMetrics: async (request) => {
+    const managementToken = request.headers.authorization.split(" ")[1];
+
+    // TODO: Validate serviceName and functionName
+    // We don't currently store and service/function names in the database to validate against.
```

Comment:
> We don't currently store and service/function names in the database to validate against.

Sorry, what do you mean by this?

Reply:
Ahh sorry this doesn't make much sense. I mean we don't have a record of the "registered" services / functions other than values we can find in the jobs table.

I.e no service definition

Reply:
Ah got it. Thanks!